Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -670,6 +778,12 @@ public function removeTrailingWhitespace($index, array $opts = array())
      */
     public function setCode($code)
     {
+        // No need to work when the code is the same.
+        // That is how we avoid a lot of work and setting changed flag.
+        if ($code === $this->generateCode()) {
+            return;
+        }
```

Comment:
I am not sure, didn't test it on 1.x line, and it wont give us any features/bugfixes on 1.x line.
We need it if we will starting to set `changed` flag.